app/controllers/concerns/list_actions.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+# Common sample actions
```

Comment:
Comment is out of date.

Reply:
Fixed in f781876c5c17b14660db3619e035a4ff5bb3d3ec